docs/settings/security-settings.asciidoc

```diff
@@ -30,6 +30,150 @@ You do not need to configure any additional settings to use the
 
 |===
 
+[float]
+[[authentication-security-settings]]
+==== Authentication security settings
+
+You configure authentication settings in the `xpack.security.authc` namespace in `kibana.yml`.
+
+For example:
+
+[source,yaml]
+----------------------------------------
+xpack.security.authc:
+    providers:
+      basic.basic1: <1>
+          order: 0 <2>
+          ...
+
+      saml.saml1: <3>
+          order: 1
+          ...
+  
+      saml.saml2: <4>
+          order: 2
+          ...
+  
+      pki.realm3:
+          order: 3
+          ...
+    ...
+----------------------------------------
+<1> Specifies the type of authentication provider (for example, `basic`, `token`, `saml`, `oidc`, `kerberos`, `pki`) and the provider name. This setting is mandatory.
+<2> Specifies order of the provider in the authentication chain and at the Login Selector UI. This setting is mandatory.
+<3> Specifies settings for the SAML authentication provider with a `saml1` name.
+<4> Specifies settings for the SAML authentication provider with a `saml2` name.
+
+The valid settings in `xpack.security.authc.providers` namespace vary depending on the authentication provider type. For more information, see <<kibana-authentication>>.
+
+[float]
+[[authentication-provider-settings]]
+===== Settings valid for all authentication providers
+
+[cols="2*<"]
+|===
+| `xpack.security.authc.providers.`
```

Comment:
__note:__ the setting name is super long and it leaves a tiny space for the setting description itself. I didn't manage to find a better way to make it readable. Another alternative I was considering is omitting `xpack.security.authc.providers.` prefix completely and explaining that all settings in this section are assumed to have one. But it didn't look well with the rest of the settings we describe in this file.

Happy to hear any suggestions!

Reply:
I wouldn't worry about the spacing issues in the tables. Gail is changing the tables into definition lists, so the spacing issue should work itself out.